distribution/run.yaml

```diff
@@ -20,19 +20,11 @@ providers:
       max_tokens: ${env.VLLM_MAX_TOKENS:=4096}
       api_token: ${env.VLLM_API_TOKEN:=fake}
       tls_verify: ${env.VLLM_TLS_VERIFY:=true}
-  - provider_id: ${env.AWS_ACCESS_KEY_ID:+bedrock}
+  - provider_id: ${env.AWS_BEARER_TOKEN_BEDROCK:+bedrock}
     provider_type: remote::bedrock
     config:
-      aws_access_key_id: ${env.AWS_ACCESS_KEY_ID:=}
-      aws_secret_access_key: ${env.AWS_SECRET_ACCESS_KEY:=}
-      aws_session_token: ${env.AWS_SESSION_TOKEN:=}
-      region_name: ${env.AWS_DEFAULT_REGION:=}
-      profile_name: ${env.AWS_PROFILE:=}
-      total_max_attempts: ${env.AWS_MAX_ATTEMPTS:=}
-      retry_mode: ${env.AWS_RETRY_MODE:=}
-      connect_timeout: ${env.AWS_CONNECT_TIMEOUT:=60}
-      read_timeout: ${env.AWS_READ_TIMEOUT:=60}
-      session_ttl: ${env.AWS_SESSION_TTL:=3600}
+      api_key: ${env.AWS_BEARER_TOKEN_BEDROCK:=}
+      region_name: ${env.AWS_DEFAULT_REGION:=us-east-2}
```

Comment:
This is a separate logical change, altering the default AWS region. The commit message and PR does not mention this - is it intentional?